### PR TITLE
ReminderTriggerRepository

### PR DESCRIPTION
**Pull request by jeffreysilver**

### CODE REVIEW

File: stopcovid/status/reminder_triggers.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    Column("drill_instance_id", UUID, nullable=False, index=True),
+    Column("prompt_slug", String, nullable=False),
+    # Do not remind for same prompt on drill twice
+    UniqueConstraint("drill_instance_id", "prompt_slug"),
+)
+
+
+Index(
+    "drill_instance_id_prompt_slug_index", "drill_instance_id", "prompt_slug",
+)
```
Comment (pdsarin) [inline stopcovid/status/reminder_triggers.py:36]: I believe that this is redundant with the uniqueness constraint.

Comment (pdsarin) [reply stopcovid/status/reminder_triggers.py:36]: and even it if isn't, it needs to be within the Table definition or it won't get created.

File: stopcovid/status/reminder_triggers.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+                for value in values:
+                    stmt = insert(reminder_triggers).values(
+                        id=str(value.id),
+                        drill_instance_id=str(value.drill_instance_id),
+                        prompt_slug=value.prompt_slug,
+                    )
+                    try:
+                        self.engine.execute(stmt)
+                    except IntegrityError:
+                        pass
```
Comment (pdsarin) [inline stopcovid/status/reminder_triggers.py:77]: when would this happen?

Comment (jeffreysilver) [reply stopcovid/status/reminder_triggers.py:77]: This is me trying to gracefully handle retries. I dont want the stream to get blocked if we try to rewrite something that already exists

Comment (pdsarin) [reply stopcovid/status/reminder_triggers.py:77]: cool. 🚢 though I'd log a warning here

(we should turn on logging :) )

**Pull request merged**